data/mods/BombasticPerks/perkdata/frankenstein.json

```diff
@@ -0,0 +1,94 @@
+[
+  {
+    "type": "recipe",
+    "activity_level": "NO_EXERCISE",
+    "result": "meat_hunk",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MEDICAL",
+    "skill_used": "firstaid",
+    "autolearn": false,
+    "difficulty": 5,
+    "skills_required": [ "electronics", 4 ],
+    "time": "240 m",
+    "qualities": [ { "id": "SEW", "level": 2 }, { "id": "SAW_M", "level": 2 }, { "id": "JUMPSTART", "level": 1 } ],
+    "components": [ [ [ "meat_tainted", 180 ] ], [ [ "bone_tainted", 80 ] ], [ [ "pipe", 2 ] ] ]
+  },
+  {
+    "id": "meat_hunk",
+    "type": "TOOL",
+    "name": { "str": "homunculus", "str_pl": "homunculi" },
+    "description": "A stitched together monstrosity of meat and bone, with limbs as wide as trash cans.  This one is still sleeping it would require a masters touch to bring it to life.",
```

Comment:
```suggestion
    "description": "A stitched together monstrosity of meat and bone, with limbs as wide as trash cans.  This one is still sleeping it would require a master's touch to bring it to life.",
```